### ISSUE TITLE
[1.16.5 Bug] Copper ore has wrong loots when mined by fortune

### ISSUE THREAD

**Issue by GoldenTotem** [ItsBlackGear/CavesAndCliffsBackport#62]
Minecraft 1.16.5 Forge 36.2.35 CavesAndCliffsBackport 7.2.0

Copper ore may drop less raw copper than normal.
In my test, I have break 200 copper ores with fortune III pickaxe. Expecting drop 3.5 * 200 * 2.2 = 1540 raw coppers. But actually drop about 1000 raw coppers. I can confirm that this is no accident because I have tested many time, and in 1.20.1 vanilla, that value is about 1540, which has no problem.
It may caused by the loot modifier raw_ore, apply_bonus function.

**Comment by ItsBlackGear** [comment]
yup! i've decided to revamp the system entirely, now it's working as intended for the upcoming update, which is most likely going to release this month.
